City-of-Helsinki/kukkuu-admin
# KK-633 | Add event group publish button

## Description

While waiting for the API endpoint to be released, I improved the test coverage of this application by adding browser tests to the event feature and by adding unit tests for some occurrence feature functions.

The main purpose of this PR is to implement the publish button functionality for event groups. After this PR the button should work.

## Context

[KK-633](https://helsinkisolutionoffice.atlassian.net/browse/KK-633)

## How Has This Been Tested?

I've added a browser test that checks that event groups can be published.

## Manual Testing Instructions for Reviewers

As a logged in user

1. Access even group list
1. Add event group
1. Select created group
1. Publish it
1. Expect it to be show as published in the events and event groups list
1. Select the created group
1. Expect publish button to be hidden
